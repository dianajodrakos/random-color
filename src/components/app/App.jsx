import React from 'react';
import Display from '../random-color/Display';

export default function App() {
  return <>
    <Display />
  </>;
}
