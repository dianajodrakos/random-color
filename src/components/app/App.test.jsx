import React from 'react';
import { render, cleanup, screen } from '@testing-library/react';
import App from './App';

describe('App component', () => {
  afterEach(() => cleanup());
  it('renders App', () => {
    const { asFragment } = render(<App />);

    const div = screen.getByRole('main');
    const p = screen.getByText('HELLO');

    expect(asFragment()).not.toBeEmptyDOMElement;
    expect(div).toContainElement(p);
  });
});
