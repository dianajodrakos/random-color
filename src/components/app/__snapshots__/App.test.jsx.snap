// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<DocumentFragment>
  <div>
    <div
      class="randomColor"
      style="background: rgb(117, 239, 213); color: rgb(119, 232, 67); border-color: #30f90c;"
    >
      <p>
        HELLO
      </p>
      <p>
        WORLD
      </p>
    </div>
  </div>
</DocumentFragment>
`;
